### PR TITLE
Bugs fixed in create exam page.

### PR DESCRIPTION
**Pull request by GcvKishore**

### CODE REVIEW

File: accounts/views.py [hunk 1]
```diff
@@ -80,6 +80,5 @@ def instructorSignUp(request):
         form = SignUpForm()
         return render(request, 'accounts/instructor-sign-up.html')
 
-
 def signIn(request):
-    return None
+    return None
```
Comment (Vedvund) [inline accounts/views.py]: You can remove this function. I guess it was added by mistake

**Review by Vedvund — CHANGES_REQUESTED** [review]
In case of error, All the fields are getting reset... It's better if we can display the previous values entered by the user...
Screenshot from 2022-04-11 17-53-02

The user can't see what he entered before...
Screenshot from 2022-04-11 17-53-08

**Review by Vedvund — APPROVED** [review]

**Review by priyushasunkara — APPROVED** [review]
Working as expected

**Review by Amulya301 — APPROVED** [review]
Validated fields with errors in create exam page.

**Pull request merged**